services/core-web/README.md

```diff
@@ -135,6 +136,7 @@ The linting rules are built on the Airbnb configuration. Contributors should
 ensure that they are not introducing linting errors into the codebase with
 their changes. Modern text editors, such as VS Code, will indicate errors.
 See the usage guide below for more information on the linting CLI options.
+// Generally we rely on the husky hooks to lint files at commit/push. Running npm run lint in core-web produces an error about import/no-cycle rule
```

Comment:
Note: let's get rid of tests as part of husky hooks (they're fine to run in PRs), and then do the linting. Possibly combine into a single PR